Python/ds/Sparse_matrix_Operation.py

```diff
@@ -1,115 +1,165 @@
 """
 
-Input:
+input:
 
-    Enter the number of rows: 3
+enter value of row: 3
 
-    Enter the number of columns: 3
+0 1 2
 
-    Enter the entries row wise:
-    1
-    1
-    0
-    0
-    0
-    0
-    0
-    2
-    0
+0 0 5
 
-Output:
-
-    Given matrix is a sparse matrix
-    [0, 0, 2]
-    [0, 1, 1]
-    [1, 1, 2]
-
-
-Time Complexity:O(noOfRows*noOfcolumns)
-Space Complexity:O(1)
+1 0 0
```

Comment:
Add description of the program at the top in commented form.
Add sample output, space and time complexity too and shift this doc string to the end of the program.